omero/developers/Web/CreateApp.rst

```diff
@@ -11,144 +11,43 @@ Getting set up
-
-.. note::
-
-    Here we use single quotes around double quotes.
-
-::
-
-    $ omero config append omero.web.apps '"minimal_webapp"'
-
-Now you can restart your omero-web server and go to
-`http://localhost:4080/minimal_webapp/ <http://localhost:4080/minimal_webapp/>`_
-in your browser.
-You should be redirected to the login screen and then back to the ``minimal_webapp``
-page which will display your Name and list your Projects.
-
-Run your app with OMERO.web in a Docker container
--------------------------------------------------
-
-The following walk-through uses `omero-web-docker <https://github.com/ome/omero-web-docker/>`_
-to run the app. Here we add ``minimal_webapp`` to the installed apps and map the
-app directory to the ``site-packages`` directory in the Docker instance so that
-python can import our ``minimal_webapp`` module.
-
-::
-
-    # You need to be in the project directory for this to work.
-    # cd omero-web-apps-examples/
-
-    # The OMERO server we want to connect to.
-    $ host=demo.openmicroscopy.org
-
-    # Path to the python module for the app.
-    $ appdir=$(pwd)/minimal-webapp/minimal_webapp
-
-    # Location within Docker instance we want to link the app, so it can be imported.
-    $ docker_appdir=/opt/omero/web/venv3/lib/python3.6/site-packages/minimal_webapp
-
-    # This example config file installs "minimal_webapp". See the file for more details.
-    $ config=$(pwd)/config.omero
-
-    # Location within Docker instance we want to mount the config.
-    $ docker_config=/opt/omero/web/config/config.omero
-
-    # Run docker container.
-    $ docker run -it --rm -e OMEROHOST=$host -p 4080:4080 -v $appdir:$docker_appdir -v $config:$docker_config openmicroscopy/omero-web-standalone
-
-This will run Docker in the foreground, showing the output in your terminal and allowing you to
-kill the container with Ctrl-C. You should see the following lines in the output, indicating
-that OMERO.web is starting and the static files from your app are being included.
-
-::
-
-    ...
-    Copying '/opt/omero/web/venv3/lib/python3.6/site-packages/minimal_webapp/static/minimal_webapp/app.css'
-    Copying '/opt/omero/web/venv3/lib/python3.6/site-packages/minimal_webapp/static/minimal_webapp/app.js'
-    ...
-    Starting OMERO.web...
-
-Now go to `http://localhost:4080/minimal_webapp/ <http://localhost:4080/minimal_webapp/>`_
-in your browser.
-You should be redirected to the login screen and then back to the ``minimal_webapp``
-page which will display your Name and list your Projects.
+Please follow the instructions there to install `cookiecutter`, create your app
+and run it within your `omero-web` python environment.
 
 Exploring the app
 -----------------
 
-The `minimal_webapp code <https://github.com/ome/omero-web-apps-examples/tree/master/minimal-webapp>`_
-is well-documented to explain how the app is working.
+The newly created app template is well-documented to explain how the app is working.
```

Comment:
Where is it well-documented ? 
If kind of "inside itself", then maybe

``Follow the comments in the created app template to understand how the app is working.`` 
would seem more plausible to me

Reply:
clarified in d3c5029d